### ISSUE TITLE
Release code for Context-Aware Pseudo-Label Scoring on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [wuyuanli60-svg/Context-Aware-Pseudo-Label-Scoring-for-Zero-Shot-Video-Summarization#1]
Hi @wuyuanli60-svg 🤗

Niels here from the open-source team at Hugging Face. I discovered your work on Arxiv and was wondering whether you would like to submit it to [hf.co/papers](https://hf.co/papers) to improve its discoverability.If you are one of the authors, you can submit it at https://huggingface.co/papers/submit.

The paper page lets people discuss about your paper and lets them find artifacts about it (your models, datasets or demo for instance), you can also claim
the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I noticed that your abstract mentions code is available at your GitHub repository (https://github.com/wuyuanli60-svg/Context-Aware-Pseudo-Label-Scoring-for-Zero-Shot-Video-Summarization), but it appears the repository is not yet populated with the code.

It'd be great to make the code for your "Context-Aware Pseudo-Label Scoring for Zero-Shot Video Summarization" framework available on the 🤗 hub once it's ready, to improve its discoverability/visibility. You could host the code as a Hugging Face Space, potentially with a Gradio demo, which would make your work immediately interactive and accessible. We can add tags so that people find them when filtering.

We can provide a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants), which gives you A100 GPUs for free, to host your demo.

If you eventually introduce any specific pre-trained models or new datasets as part of your framework, we would also be happy to help host them on the Hub:
*   **Models:** See here for a guide: https://huggingface.co/docs/hub/models-uploading.
*   **Datasets:** See here for a guide: https://huggingface.co/docs/datasets/loading.

Let me know if you're interested/need any help regarding this!

Cheers,

Niels
ML Engineer @ HF 🤗
